docs/admin/deployment/onboarding_admins.rst

```diff
@@ -31,60 +31,56 @@ To set up AW2, follow these steps:
-9. Open the file manager (**Applications ▸ Accessories ▸ Files**).
-10. Insert AW1. It should show up in the list of storage devices in the file manager under
+9. Reboot AW2, unlock its persistent volume, and set an admin password on the welcome screen.
+10. Open the file manager (**Applications ▸ Accessories ▸ Files**).
+11. Insert AW1. It should show up in the list of storage devices in the file manager under
     a label like "7.0 GB Encrypted". Click the label and enter the drive
     password when prompted to unlock it.
-11. In a terminal, type the following command:
+12. In a terminal, type the following command:
 
     ``rsync -a /media/amnesia/TailsData/Persistent/securedrop ~/Persistent``
 
     This will copy *only* the ``securedrop`` directory from AW1 to AW2.
-12. Generate a new keypair on AW2 using the following command:
+13. Generate a new keypair on AW2 using the following command:
 
     ``ssh-keygen -t rsa -b 4096``
 
     When prompted, store the keypair in the default location.
-13. Shut down AW2.
-14. Boot AW1, unlock its persistent volume, and set an admin password on
-    the welcome screen.
-15. Open the file manager (**Applications ▸ Accessories ▸ Files**).
-16. Insert AW2 and unlock it.
-17. In a terminal, type the following commands to authorize the newly created SSH keypair
-    on your servers:
-
-    ``ssh-copy-id -i /media/amnesia/TailsData/openssh-client/id_rsa.pub app``
-    ``ssh-copy-id -i /media/amnesia/TailsData/openssh-client/id_rsa.pub mon``
-18. Log into the *Journalist Interface* using your admin credentials, and create
-    a new user account with admin rights. Record its passphrase securely;
-    you will add it to the password manager on AW2.
-
-    (You will need to on-board the new admin's 2FA device to complete this step.
-    If this is not possible yet, you can defer it until later.)
-19. Shut down AW1.
-20. Boot AW2, unlock its persistent volume, and set an admin password
-    on the welcome screen.
-21. Boot into AW2 and run the command ``./securedrop-admin tailsconfig`` in
-    ``~/Persistent/securedrop``.
+14. Run the command ``./securedrop-admin tailsconfig`` in ``~/Persistent/securedrop``.
 
     This will set up desktop shortcuts and SSH access.
-22. Confirm that you are able to access ``mon`` and ``app`` via SSH (``ssh app`` and ``ssh mon``).
-23. Confirm that you are able to access the *Source Interface* and the *Journalist
+15. Insert AW1. It should show up in the list of storage devices in the file manager under
+    a label like "7.0 GB Encrypted". Click the label and enter the drive
+    password when prompted to unlock it.
+16. In a terminal, type the following commands to authorize the newly created SSH keypair
+    on your servers:
+    ``ssh-add``
+    ``ssh-add /media/amnesia/TailsData/openssh-client/id_rsa``
+    ``ssh-copy-id app``
+    ``ssh-copy-id mon``
+    ``ssh-add -D``
+17. Confirm that you are able to access ``mon`` and ``app`` via SSH (``ssh app`` and ``ssh mon``).
```

Comment:
I would unmount AW1 before this step just in case ssh-add -D doesn't do the needful. Having the AW1 key completely unavailable would be cool. Also consider making the check something like "ssh app hostname" or even making it a one-liner that runs a similar remote command and true/falses based on the output. No need to drop folks into a shell.

Reply:
See <https://github.com/freedomofpress/securedrop-docs/pull/484#discussion_r1344877654>.